### DISCUSSION TITLE
Pumping crude oil using create

### DISCUSSION THREAD

**Discussion by yeaxsam** [TeamPneumatic/pnc-repressurized#1249] [Q&A]
Why am I not able to pump out crude oil using create hoses and pumps? The hose pulley just hovers over the liquid and cant be lowered further.

Is there any config that I'd be able to change so that the crude oil is able to be pumped out of the ground via Create?

**Comment by desht** [comment]
No idea, crude oil is just a normal liquid as far as mods are concerned.  Maybe ask in the PNC and/or Create discords.

**Reply by yeaxsam** [reply]
The oil behaves almost like its a solid block
meaning I can't even place anything in the pond itself 
![Createbug](https://github.com/TeamPneumatic/pnc-repressurized/assets/149759243/3a8b158f-d3a4-4d2e-a8b1-95f980d531c1)

The hose pulley doesn't go into it no matter what I try

**Reply by desht** [reply]
That's a known problem, will be fixed in the next release, coming in the next day or two. It might also fix the problem with oil not being pumpable by create.
